app/views/admin/review_applicant.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="container-large">
+    <div class="row">
+        <div class="modal-content col s12">
+            <h4><%= @applicant.fullname%></h4>
+            <ul class="li_item">
+                <% @applicant.skillsets.each do |skillset| %>
+                <li class="trade">
+                    <%= skillset.name %>&nbsp
+                </li>
+                <% end %>
+            </ul>
+
+        </div>
+        <div class="modal-content col s10">
+            <% question = "What is your name" %>
+            <% answer = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis
+            aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum." %>
+            <ul>
+                <% 5.times do |qsn| %>
+                <li><%= question %>
+                </li>
+                <li class="answers">
+                    <%= answer %>
+                </li>
+                <% end %>
+            </ul>
+
+            <form action="<%= update_applicant_path %>" method="post">
```

Comment:
Following my comment on the controller, you can use Rails form helpers here with the `update` action.